core/src/main/java/pl/agh/edu/ui/component/label/CustomLabel.java

```diff
@@ -1,17 +1,32 @@
 package pl.agh.edu.ui.component.label;
 
+import static pl.agh.edu.ui.utils.SkinColor.ColorLevel._300;
+import static pl.agh.edu.ui.utils.SkinColor.ColorLevel._500;
+import static pl.agh.edu.ui.utils.SkinColor.ColorLevel._700;
+import static pl.agh.edu.ui.utils.SkinColor.ColorLevel._900;
+import static pl.agh.edu.ui.utils.SkinColor.GRAY;
+import static pl.agh.edu.ui.utils.SkinColor.SECONDARY;
 import static pl.agh.edu.ui.utils.SkinSpecialColor.TRANSPARENT;
 
 import com.badlogic.gdx.graphics.Color;
 import com.badlogic.gdx.graphics.g2d.Batch;
 import com.badlogic.gdx.graphics.g2d.NinePatch;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.InputListener;
 import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
 import com.badlogic.gdx.scenes.scene2d.utils.NinePatchDrawable;
+import com.badlogic.gdx.utils.Null;
 
 import pl.agh.edu.ui.GameSkin;
+import pl.agh.edu.ui.audio.SoundAudio;
+import pl.agh.edu.ui.utils.SkinColor;
 
 public class CustomLabel extends Label {
 	private Color underscoreColor = TRANSPARENT.getColor();
+	private boolean isDisabled = true;
+	private SkinColor baseColor = SECONDARY;
 
 	public CustomLabel(String font, String backgroundPatch) {
 		this(font);
```

Comment:
jak robimy refactor to odrazu był ustawił skina jako pole i używał tylko jego bo już w kilku miejscach pojawią nam się wywoływanie instancji